### ISSUE TITLE
Error While Building V-EZ

### ISSUE THREAD

**Issue by harryisgamer** [GPUOpen-LibrariesAndSDKs/V-EZ#41]
I am building V-EZ, and I keep getting errors when building.
Using VS2017
Vulkan SDK 1.1.82

Build Output: https://pastebin.com/C5qFpRZc

**Comment by soconne** [comment]
You're getting these build errors on a clean pull from github?  No changes were made to ObjectLookup.h or ObjectLookup.cpp?

**Comment by myuniquename** [comment]
Yes, the issue is reproducible on a clean V-EZ version I've got form git. 
Please note, issue is reproducible when you're compiling win32 configuration. x64 compiles successfully.

It looks like microsoft compiler thinks VkImage having same type as (e.g.) VkFence since both of them defined as uint64_t at \VulkanSDK\Include\vulkan\vulkan_core.h:77 using VK_DEFINE_NON_DISPATCHABLE_HANDLE macro.

Tested in VS2017 and VS2015. Please, let me know if you need more info.

**Comment by myuniquename** [comment]
So, it looks like nobody even tried to compile 32bit version of V-EZ before. So, I've hacked a version for my needs quickly. It's available here https://github.com/myuniquename/V-EZ. SimpleQuad sample works fine.

**Comment by harryisgamer** [comment]
Ok thanks, sorry for the slow response from me.

**Comment by soconne** [comment]
There was never any intention of supporting 32-bit compilations as both AMD and our competitor no longer offer 64-bit drivers.

**Comment by myuniquename** [comment]
> There was never any intention of supporting 32-bit compilations as both AMD and our competitor no longer offer 64-bit drivers.

Well, I understand the idea of not supporting 32 bit compilation in 2018. However, IMHO it has to be clearly stated in project description. Because 64 bit OS using 64 bit driver still can run 32 bit application.

**Comment by cappah** [comment]
I tried to build it in CMake 3.13.2. When I pushed "Configure" and set the generator to Visual Studio 15 2017 Win64,  the message displayed:

> CMake Warning (dev) in CMakeLists.txt:
  No cmake_minimum_required command is present.  A line of code such as cmake_minimum_required(VERSION 3.13) should be added at the top of the file.

The CMakeLists.txt is set to VERSION 3.8 as the minimum.

After clicking "Generate," the message displayed:
https://pastebin.com/vSyaSUpy

Then when trying to build the project in VS2017:
https://pastebin.com/bdeA8uBs

If it matters, my computer uses Nvidia GeForce GTX and I have Vulkan 1.1.92.1
